bind/flag.go

```diff
@@ -126,8 +126,15 @@ func DenyDomains(fs *pflag.FlagSet, cfg *[]ruleset.RegexpListItem) {
 	fs.Var(anyflag.NewSliceValue[ruleset.RegexpListItem](*cfg, cfg, ruleset.ParseRegexpListItem),
 		"deny-domains", "[-]<regexp>,..."+
 			"Deny requests to the specified domains. "+
-			"Prefix domains with '-' to exclude requests to certain domains from being denied.",
-	)
+			"Prefix domains with '-' to exclude requests to certain domains from being denied.")
+}
+
+func DirectDomains(fs *pflag.FlagSet, cfg *[]ruleset.RegexpListItem) {
+	fs.Var(anyflag.NewSliceValue[ruleset.RegexpListItem](*cfg, cfg, ruleset.ParseRegexpListItem),
+		"direct-domains", "[-]<regexp>,..."+
+			"Directly connect to the specified domains without using the upstream proxy. "+
```

Comment:
How about `Directly connect` -> `Connect directly`? It sounds odd to start with an adverb here.

Reply:
Yes.